pkg/formula/input/input.go

```diff
@@ -51,30 +61,100 @@ func inputConditionVariableExistsOnInputList(variable string, inputList formula.
 	return false
 }
 
+func containsSubstring(s string, substr string) bool {
+	return strings.Contains(s, substr)
+}
+
+func valueContainsAny(inputType string, value string, input string) bool {
+	splitInput := strings.Split(input, "|")
+	if inputType == MultiselectType {
+		splitValue := strings.Split(value, "|")
+		for _, i := range splitInput {
+			if containsArray(splitValue, i) {
+				return true
+			}
+		}
+	} else {
+		for _, i := range splitInput {
+			if containsSubstring(value, i) {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func valueContainsAll(inputType string, value string, input string) bool {
+	splitInput := strings.Split(input, "|")
+	if inputType == MultiselectType {
+		splitValue := strings.Split(value, "|")
+		for _, v := range splitInput {
+			if !containsArray(splitValue, v) {
+				return false
+			}
+		}
+	} else {
+		for _, v := range splitInput {
+			if !containsSubstring(value, v) {
+				return false
+			}
+		}
+	}
+	return true
+}
+
+func valueContainsOnly(inputType string, value string, input string) bool {
+	if inputType == MultiselectType {
+		splitInput := strings.Split(input, "|")
+		splitValue := strings.Split(value, "|")
+		if len(splitValue) != len(splitInput) {
+			return false
+		}
+		for _, v := range splitInput {
+			if !containsArray(splitValue, v) {
+				return false
+			}
+		}
+	} else {
+		if !strings.EqualFold(value, input) {
+			return false
+		}
+	}
+	return true
```

Comment:
I think you can simplify this line
```suggestion
	} else {
		return strings.EqualFold(value, input)
	}
```

Reply:
done